app/javascript/packages/document-capture/components/in-person-prepare-step.jsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Alert,
+  Icon,
+  IconList,
+  IconListContent,
+  IconListIcon,
+  IconListItem,
+  IconListTitle,
+  PageHeading,
+  ProcessList,
+  ProcessListHeading,
+  ProcessListItem,
+} from '@18f/identity-components';
+import { FormStepsButton } from '@18f/identity-form-steps';
+import { useI18n } from '@18f/identity-react-i18n';
+import InPersonTroubleshootingOptions from './in-person-troubleshooting-options';
+
+/**
+ * @typedef InPersonPrepareStepValue
+ *
+ * @prop {Blob|string|null|undefined} inPersonPrepare InPersonPrepare value.
+ */
+
+/**
+ * @param {import('@18f/identity-form-steps').FormStepComponentProps<InPersonPrepareValue>} props Props object.
+ */
+function InPersonPrepareStep({ errors = [] }) {
+  const { t } = useI18n();
+  const error = errors.find(({ field }) => field === 'in-person-prepare')?.error;
+
+  return (
+    <>
+      <Alert type="success" className="margin-bottom-4">
+        {t('in_person_proofing.body.prepare.alert_selected_post_office', { name: 'EASTCHESTER' })}
+      </Alert>
+      <PageHeading>{t('in_person_proofing.headings.prepare')}</PageHeading>
+
+      <p>{t('in_person_proofing.body.prepare.verify_step_about')}</p>
+
+      <ProcessList className="margin-bottom-4">
+        <ProcessListItem>
+          <ProcessListHeading unstyled>
+            {t('in_person_proofing.body.prepare.verify_step_enter_pii')}
+          </ProcessListHeading>
+        </ProcessListItem>
```

Comment:
Similarly I wonder if we could collapse some components here:

```suggestion
        <ProcessListItem heading={t('in_person_proofing.body.prepare.verify_step_enter_pii') />
```

Reply:
addressed in 027b25e3f8eefea84e70645f48a53f19d5a21aba